### PR TITLE
Add params export function

### PR DESCRIPTION
**Pull request by rernst**
Add ExportParams process, modules PR: https://github.com/UMCUGenetics/NextflowModules/pull/56

### CODE REVIEW

File: MipFingerprint.nf [hunk 1]
```diff
@@ -113,6 +116,7 @@ process VersionLog {
     tag {"VersionLog"}
```
Comment (FiniDG) [inline MipFingerprint.nf:116]: is there a reason that the parameter ${analysis_id}"} is missing in only this pull request (MIP, compared to the WGS and WES pull requests of the same function)

**Review by FiniDG — APPROVED** [review]
reviewed along with pull requests of WESnr44 and WGSnr9 with same function added.
not further comments, except for the small differences between the three pull requests. see comments in those pull requests if applicable.
Approved

File: MipFingerprint.nf (hunk 1 above)
Comment (rernst) [reply MipFingerprint.nf]: No particular reason, probably changed in WES / WGS after implementing it in this workflow. Added it for consistency.

**Review by FiniDG — APPROVED** [review]
approved after change and explanation

**Pull request merged**